docker-compose.yml

```diff
@@ -22,17 +22,22 @@ services:
     ports:
       - "5306:3306"
     volumes:
+      - ./.data/mysql:/var/lib/mysql
       - ./mysql:/docker-entrypoint-initdb.d/:ro
     container_name: student_dashboard_mysql
   web:
     build: .
     command: bash -c "./start.sh"
     volumes:
       - .:/code
+      # use the container's node_modules folder (don't override)
```

Comment:
still keep line 32?

Reply:
This is a little trick for development environments. By mounting the all the host project files within the container (in this case `.:/code`) you can have your local changes be available to the container without rebuilding. 

This however can be alloying with certain subfolders containing dependencies (in this case `/code/node_modules`). What you can then do to get around having to build this folder locally is add a volume for the subfolder without any host (in this case `/code/node_modules/`) to basically tell docker to use the container's version of the folder instead of the host's